src/plugins/mission_raw/mission_raw_impl.cpp

```diff
@@ -252,6 +252,9 @@ void MissionRawImpl::process_mission_item_int(const mavlink_message_t& message)
     }
 
     auto new_item = std::make_shared<MissionRaw::MavlinkMissionItemInt>();
+    if (!new_item) {
+        return;
+    }
```

Comment:
Has this ever happened? How did you find this?

Reply:
One of the platforms threw warnings as errors if this wasn't checked... I forgot which - I have to check.

Reply:
I removed this change.